Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -110,7 +59,7 @@ if($deployFilesInParallel -eq "false")
         $fdqn = $resource.Name
```

Comment:
is there supposed to be an FQDN property bag? or we will implement that later to pick it up from?